### PR TITLE
pythonPackage.foxdot: init at 0.8.1

### PR DESCRIPTION
**Pull request by MrMebelMan**
###### Motivation for this change

Just adding the package that I was badly missing.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by MrMebelMan** [comment]
Added `supercollider` to `propagatedBuildInputs`, added a comment explaining why tests are disabled and fixed the license :pray:

**Comment by mmahut** [comment]
@GrahamcOfBorg build foxdot

**Comment by mmahut** [comment]
@GrahamcOfBorg build pythonPackages.foxdot python3Packages.foxdot

**Comment by MrMebelMan** [comment]
Removed the conflicting maintainer commit & rebased master

**Comment by mmahut** [comment]
@GrahamcOfBorg build pythonPackages.foxdot python3Packages.foxdot

File: pkgs/development/python-modules/foxdot/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi, tkinter, supercollider }:
+
+buildPythonPackage rec {
+  pname = "FoxDot";
+  version = "0.7.30";
```
Comment (mmahut) [inline pkgs/development/python-modules/foxdot/default.nix]: ``0.8.1`` is available.

Comment (MrMebelMan) [reply pkgs/development/python-modules/foxdot/default.nix]: updated :+1:

**Comment by mmahut** [comment]
@GrahamcOfBorg build pythonPackages.foxdot python3Packages.foxdot

**Pull request merged**